packages/connector-node-v1/src/api.js

```diff
@@ -74,31 +43,24 @@ async function getParentsForId(options, id, result = []) {
   }
 
   let parent = await getResourceById(options, parentId);
-  return await getParentsForId(options, resource.parentId, [parent].concat(result));
+  return getParentsForId(options, resource.parentId, [parent].concat(result));
```

Comment:
Extra `getResourceById()` is not needed when 2nd argument of `getParentsForId()` becomes **parent** instead of **resource.parentId**